app/scripts/modules/google/src/serverGroup/configure/serverGroupConfiguration.service.js

```diff
@@ -369,7 +369,7 @@ module.exports = angular
         const regions = command.backingData.credentialsKeyedByAccount[command.credentials].regions;
         if (_.isArray(regions)) {
           filteredData.zones = _.find(regions, { name: command.region }).zones;
-          filteredData.truncatedZones = _.takeRight(filteredData.zones.sort(), 3);
+          filteredData.truncatedZones = filteredData.zones.slice().sort();
```

Comment:
should this variable be renamed since the list is no longer truncated?

Reply:
i've renamed these & tested that pipelines saved before/after the rename continue to render as expected.